src/js/controllers/mapController.ts

```diff
@@ -63,11 +85,54 @@ export class MapController {
               title,
               opacity,
               visible,
-              definitionExpression
+              definitionExpression,
+              group: 'webmap'
             });
           });
+
           store.dispatch(allAvailableLayers(mapLayerObjects));
 
+          this.getMoreLayers().then(res => {
+            const { appState } = store.getState();
+
+            const resourceLayerObjects: LayerProps[] = [];
+
+            res.forEach((apiLayer: RemoteDataLayer) => {
+              let resourceId;
+              let resourceTitle;
+              let resourceOpacity = 1; //TODO: Make this dynamic
```

Comment:
To determine layer opacity we have to go through up to 3 steps: 1) Check if we have it defined in the resources.js 2) check if api response came back with opacity defined, 3) if both have nothing default to `1`

This is a little ugly but could work:
```
              function determineLayerOpacity() {
                //Try the resources.js predetermined opacity
                let opacity = apiLayer.dataLayer?.opacity;
                if (!opacity) {
                  //nothing in the resources to do with opacity, try the response's oapcity
                  opacity = apiLayer?.layer?.opacity;
                }
                return opacity || 1; //if all fails, default to 1
              }
```